addons/web/static/src/scss/web_calendar.scss

```diff
@@ -468,18 +468,23 @@ $o-cw-filter-avatar-size: 20px;
         th {
             padding: .7em .2em;
             width: 14%;
+
+            span {
```

Comment:
try to use '>' in such selectors to speed up the rule resolution (if you don't know how rules are applied, I can roughly explain it to you)